src/views/virtualmachines/details/tabs/diagnostic/virtual-machine-diagnostic-tab.scss

```diff
@@ -2,3 +2,17 @@
   display: flex;
   justify-content: space-between;
 }
+
+.VirtulMachineDiagnosticTabRow--expended {
+  display: inline-flex;
+  align-items: baseline;
+  margin-left: var(--pf-global--spacer--3xl);
+  svg {
+    margin-right: var(--pf-global--spacer--sm);
+  }
+}
+.pf-c-table {
+  tr.VirtulMachineDiagnosticTabRow--row {
```

Comment:
Typo in the class name: 'Virtul' -> 'Virtual'

Reply:
thanks!